app/src/main/java/q4/team1/CartItem.java

```diff
@@ -21,19 +21,16 @@ public int getQuantity(){
     return quantity;
   }
   public double getPrice(){
-    return price;
+    return this.price = this.orgPrice*this.quantity;
   }
   public double getOrgPrice(){
-    return orgPrice;
+    return this.orgPrice;
   }
-  public String setName(){
-    return name;
-  }
-  public int setQuantity(){
-    return quantity;
+  public void setName(String name){
+    this.name = name;
   }
-  public double setPrice(){
-    return price;
+  public void setQuantity(int quantity){
+    this.quantity = quantity;
```

Comment:
this can be this.quantity = quantity. then under it we can do setPrice(origPrice*quantity)

Reply:
this is to avoid having to change code in 2 places and we can change it in one